fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -1681,6 +1681,12 @@ void SimulationConfig::setDatacenters(const TestConfig& testConfig) {
 		// overwrite whatever decision we made before
 		generateFearless = testConfig.generateFearless.get();
 	}
+#ifdef NO_MULTIREGION_TEST
+	if (generateFearless) {
+		// Tests requiring multiregion cannot run if desired build does not support it.
+		throw internal_error;
```

Comment:
I guess `throw internal_error();` might be better while `throw internal_error_msg("Test requires multi-region while the flag is turned off in the build process");` would be even better.